MonoGame.Framework.Content.Pipeline/Utilities/RuntimeTypeStrings.cs

```diff
@@ -0,0 +1,20 @@
+// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
+
+namespace MonoGame.Framework.Content.Pipeline
+{
+    public static class RuntimeTypeStrings

```

Comment:
It may be useful for this to be `public` for pipeline extensions to be written against.

If so we should probably also document it.

Reply:
I intended for this to be internal. It might be useful to make it public for CP extensions that build xnbs that should be read with built-in readers.